lessons/module-3/react-class-components-prework.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Class Component Prework - Classes in JavaScript
+module: 3
+---
+
+## Learning Goals
+
+By the end of this lesson, you will be able to:
+
+* explain what a class is
+* define classes using the `class` keyword
+* create object instances
+* call methods on object instances
+
+## Vocabulary
+- `this` - a keyword with a value that changes depending on the context in which it's used
+- `class` - a special function which functionas as a template for creating objects
+- `object instance` - objects that were created from a class and contain the data and functionality defined in that class
+- `inheritance` - a mechanism which derives a class from another class; one class can inherit properties and functionality from a parent class
```

Comment:
I think I would prefer to cut the parts about inheritance to try to slim this down.  I dont think current M3 student really get this concept currently either.